### ISSUE TITLE
HVU modeling details

### ISSUE THREAD

**Issue by digbose92** [holistic-video-understanding/HVU-Dataset#9]
Hi, I am trying to replicate the performance of 3D-Resnet-18 on HVU dataset by training for scene labels. 

In the paper (https://arxiv.org/pdf/1904.11451.pdf), Table 3, the MAP is listed as 50.6%.

Is it possible to share the training details for 3D-Resnet-18? 

I am getting a MAP of 37% on 251,794 videos by training using the following configuration:

1. **Learning rate**: 1e-4

2. **Scheduler**: LR scheduler on the plateau (decrease the learning rate by 0.1 when the validation MAP does not improve for 3 epochs)

3. **Optimizer**: Adam

4. **Batch size**: 64

5. Temporal Random crop as augmentations for training followed by scaling and normalization using mean and std of kinetics dataset.

6. Center crop as augmentations for validation followed by scaling and normalization using mean and std of kinetics dataset.

Thanks in advance
